src/BenchmarkDotNet/Templates/WasmCsProj.txt

```diff
@@ -26,6 +26,8 @@
     <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
     <ValidateExecutableReferencesMatchSelfContained>false</ValidateExecutableReferencesMatchSelfContained>
     <EnableDefaultWasmAssembliesToBundle>false</EnableDefaultWasmAssembliesToBundle>
+    <!-- fix System.Runtime.CompilerServices.Unsafe doesn't support netcoreapp2.0 -->
```

Comment:
this project file should never be used for such old tfms (because WASM was not supported back then)